server/src/lib/queues/imageQueue.ts

```diff
@@ -0,0 +1,38 @@
+// import { S3 } from 'aws-sdk';
+import * as Queue from 'bull';
+import { getConfig } from '../../config-helper';
+// import Bucket from '../bucket';
+import imageProcessor from './processes/imageProcessor'; // producer
+
+const redisUrlParts = getConfig().REDIS_URL?.split('//');
```

Comment:
Style not important, could we put all of this logic in a 'getRedisOptions' function or something?